Cargo.toml

```diff
@@ -100,6 +100,7 @@ deltalake = { git = "https://github.com/splitgraph/delta-rs", branch = "fix-deci
 
 futures = "0.3"
 hex = ">=0.4.0"
+iceberg = { git = "https://github.com/apache/iceberg-rust", rev = "697a20060f2247da87f73073e8bf5ab407bd40ea" }
```

Comment:
On main we changed the iceberg-datafusion repo to `splitgraph/iceberg-rust`. I suggest updating this branch from main and also using `splitgraph/iceberg-rust` for this new dependency

Reply:
Rebased and pointed to our fork.